packages/gatsby-plugin-manifest/README.md

```diff
@@ -130,6 +134,79 @@ icons: [
 
 In the manual mode, you are responsible for defining the entire web app manifest and providing the defined icons in the [static](https://www.gatsbyjs.org/docs/static-folder/) folder. Only icons you provide will be available. There is no automatic resizing done for you.
 
+### i18n – Multilang configuration
```

Comment:
```suggestion
#### i18n – Multilang configuration
```
This whole section should be placed under "Feature Configuration" heading. This conforms to the progressive nature of the docs flow.